README.md

```diff
@@ -333,3 +348,11 @@ If you want to get involved or have ideas you'd like to chat about, we discuss t
 See the [Community Calendar](https://calendar.google.com/calendar?cid=czYzdm9lZmhwNWk5cGZsdGI1cTY3bmdwZXNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) for the schedule and meeting invitations.
 
 See the [Contributing](CONTRIBUTING.md) documentation for guidance on how to contribute.
+
+## Community Meetings
+
+The meetings happen biweekly https://meet.google.com/kzd-tzzt-efe
```

Comment:
Should we provide the calendar link to OSSF calendar instead? - https://calendar.google.com/calendar/embed?src=s63voefhp5i9pfltb5q67ngpes%40group.calendar.google.com&ctz=America%2FLos_Angeles

Reply:
Thanks, updated the calendar link.